elide-contrib/elide-test-helpers/src/main/java/com/yahoo/elide/contrib/testhelpers/graphql/elements/StringValue.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.testhelpers.graphql.elements;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * A {@link StringValue} is a string value that is quoted in its serialized form.
+ *
+ * @see EnumValue
+ */
+@RequiredArgsConstructor
+public class StringValue implements ValueWithVariable {
```

Comment:
Do we really need all these wrapper classes for the types?  Why can't we just use jackson and serialize plain old Java objects?

Reply:
Done